packages/amazonq/.changes/next-release/Bug Fix-42b2c2f0-e40e-4f97-842b-cb7ead1a010b.json

```diff
@@ -0,0 +1,4 @@
+{
+	"type": "Bug Fix",
+	"description": "fix bug where code fix line number or file was not accurate"
```

Comment:
```suggestion
	"description": "Code fix line number or file is sometimes not accurate"
```

Reply:
Thanks for including a changelog. Please review the [changelog guidelines](https://github.com/aws/aws-toolkit-vscode/blob/master/CONTRIBUTING.md#changelog).